account_partner_aged_statement_webkit/report/partner_aged_statement.mako

```diff
@@ -0,0 +1,174 @@
+## -*- coding: utf-8 -*-
+<!DOCTYPE html SYSTEM
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <style type="text/css">
+            .total{
+               font-weight:bold;
+            }
+            .break{
+                display: block;
+                clear: both;
+                page-break-after: always;
+            }
+
+            ${css}
+        </style>
+    </head>
+
+    <%!
+    def amount(text):
+        # replace by a non-breaking hyphen (it will not word-wrap between hyphen and numbers)
+        return text.replace('-', '&#8209;')
+
+    %>
+    <body>
+    %for idx, partner in enumerate(objects):
+        <%setLang(partner.lang)%>
+        %if idx > 0:
+            <div class="break"></div>
+        %endif
+        <br/>
+        <br/>
+        <%from datetime import date %>
+        ${_('Date')}: ${formatLang(str(date.today()), date=True)}
+        <br>
+        ${_('Partner')}: ${partner.name}
+        <br>
+        ${_('Subject')}: <b>${_('Overdue Statement')}</b>
+        <br/>
+        <br/>
+        %if show_message:
+            %for message_line in message(partner, company):
+                <p>
+                ${message_line}
+                </p>
+            %endfor
+        %endif
+        <br>
+        ${user.name}
+        <br>
+        <br>
+        %if (partner.credit + partner.debit == 0) :
+            <div class="title">${_('Nothing due for this partner')}</div>
+        %else:
+            <div class="title">${_('Aged Balance')}</div>
+            <br>
+        %if get_balance(partner, company):
+            <table class="basic_table" style="width: 100%;">
+                <tr>
+                    <th>${_('Not Due')}</th>
+                    <th>${_('0-30')}</th>
+                    <th>${_('30-60')}</th>
+                    <th>${_('60-90')}</th>
+                    <th>${_('90-120')}</th>
+                    <th>${_('+120')}</th>
+                    <th>${_('Total')}</th>
+                    <th>${_('Currency')}</th>
+                </tr>
+                %for l in get_balance(partner, company):
+                <tr>
+                    <td>${ l['not_due'] }</td>
+                    <td>${ l['30'] }</td>
+                    <td>${ l['3060'] }</td>
+                    <td>${ l['6090'] }</td>
+                    <td>${ l['90120'] }</td>
+                    <td>${ l['120'] }</td>
+                    <td>${ l['total'] }</td>
+                    <td>${ l['currency_name']}</td>
+                </tr>
+                %endfor
+            </table>
+        %endif
+        <br>
+        <br>
```

Comment:
These `<br>` tags aren't closed. It's not a big deal. Having open tags prevent xml parser to parse html files. If this module was to be ported to odoo8 QWeb templates, it would be better to keep mako templates xml friendly as much as possible. Also instead of using `<br>`. It would make more sense to add a margin on block elements. Line breaks might get added multiple times if content between the blocks isn't added. For example, here if the test for `get_balance` doesn't pass, you'll have 3 line breaks instead of 1 or 2..